hadoop-hdds/docs/content/interface/OzoneFS.md

```diff
@@ -57,12 +53,14 @@ Please add the following entry to the core-site.xml.
 
 This will make this bucket to be the default file system for HDFS dfs commands and register the o3fs file system type.
 
-You also need to add the ozone-filesystem.jar file to the classpath:
+You also need to add the ozone-filesystem-hadoop3.jar file to the classpath:
 
 {{< highlight bash >}}
-export HADOOP_CLASSPATH=/opt/ozone/share/ozonefs/lib/hadoop-ozone-filesystem-lib-current*.jar:$HADOOP_CLASSPATH
+export HADOOP_CLASSPATH=/opt/ozone/share/ozonefs/lib/hadoop-ozone-filesystem-hadoop3-*.jar:$HADOOP_CLASSPATH
 {{< /highlight >}}
 
+(Note: with Hadoop 2.x, use the `hadoop-ozone-filesystem-hadoop3-*.jar`)
```

Comment:
Do you mean hadoop-ozone-filesystem-hadoop2-*.jar?

Reply:
:blush: 

It seems that I lost in the forests of jar files. Thanks, fixing it.